packages/gatsby-theme-newrelic/src/components/EditPageButton.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+import Link from './Link';
+import useInstrumentedHandler from '../hooks/useInstrumentedHandler';
+import useThemeTranslation from '../hooks/useThemeTranslation';
+import { graphql, useStaticQuery } from 'gatsby';
+
+const EditPageButton = ({
+  fileRelativePath,
+  onClick,
+  instrumentation,
+  ...props
+}) => {
+  const { t } = useThemeTranslation();
+  const handleClick = useInstrumentedHandler(onClick, {
+    actionName: 'editThisPage_click',
+    component: instrumentation?.component,
+  });
+
+  const {
+    site: {
+      siteMetadata: { repository, branch },
+    },
+  } = useStaticQuery(graphql`
+    query {
+      site {
+        siteMetadata {
+          repository
+          branch
+        }
+      }
+    }
+  `);
+
+  return (
+    <Button
+      {...props}
+      as={Link}
+      to={`${repository}/blob/${branch}/${fileRelativePath || ''}`}
+      onClick={handleClick}
+    >
+      <Icon
+        name="fe-edit"
+        css={css`
+          margin-right: 0.5rem;
+        `}
+      />
+      {t('github.editPage')}
```

Comment:
I do we have a fallback string if the theme user doesn't have this set up? I don't fully remember how this stuff works.

Reply:
The theme uses its own namespace for translations. Those translations are a part of the theme itself. You can find that here: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/i18n/translations/en.json

As long as we have a definition in there, we won't need the fallback. No need for a site using the theme to have to remember to add these definitions or not since the theme ships with them.

Reply:
That being said, might be good to send off these strings to get translated soon.